config/acer-a1-724.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# DTB: From Acer_AV0K1_A1-724_RV04RC07_WW_GEN1.zip
+# https://androidfilehost.com/?fid=889764386195913456
```

Comment:
Todo: The source of official image have been found.
https://global-download.acer.com/GDFiles/OS/OS/OS_Acer_AV0K1.RV04RC07.WW.GEN1_A44K_A.zip?acerid=635817910038506597&Step1=&Step2=&Step3=A1-724&OS=ALL&LC=en&BC=ACER&SC=PA_6